lib/metasploit/framework/jtr/wordlist.rb

```diff
@@ -369,12 +369,13 @@ def mutation_keys
         # This method takes all the options provided and streams the generated wordlist out
         # to a {Rex::Quickfile} and returns the {Rex::Quickfile}.
         #
+        # @param max_len [Integer] max length of a word in the wordlist, 0 default for ignored value
         # @return [Rex::Quickfile] The {Rex::Quickfile} object that the wordlist has been written to
-        def to_file
+        def to_file(max_len)
           valid!
           wordlist_file = Rex::Quickfile.new("jtrtmp")
           each_word do |word|
-            wordlist_file.puts word
+            wordlist_file.puts max_len == 0 ? word : word[0..max_len-1]
```

Comment:
```
2.3.0 :001 > word = 'asdfghjkl'
 => "asdfghjkl" 
2.3.0 :002 > max_len = 8
 => 8 
2.3.0 :003 > word[0..max_len-1]
 => "asdfghjk" 
2.3.0 :004 > word[0...max_len]
 => "asdfghjk" 
2.3.0 :005 > 
```

;)

Reply:
you never fail to teach me new ruby EVERY PR I submit. kudos! +1